src/encoded/static/components/browse/components/FacetCharts.js

```diff
@@ -201,7 +184,7 @@ export class FacetCharts extends React.PureComponent {
         const show = this.show();
         if (!show) return null; // We don't show section at all.
 
-        const { context, debug, windowWidth, colWidthPerScreenSize, schemas, href, isFullscreen } = this.props;
+        const { context, debug, windowWidth, colWidthPerScreenSize, schemas, href, isFullscreen, initialFields } = this.props;
```

Comment:
Codacy found an issue: ['colWidthPerScreenSize' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6178047)